src/utils/cloudinary.js

```diff
@@ -0,0 +1,7 @@
+const cloudinary = require("cloudinary").v2;
```

Comment:
- Let us move the integration part to a separate file. We will use module wrapping, make our module for upload, and then export from this module function `upload`. As we talked about this before, when you made axios wrapper, it's much easier tomorrow to replace the module, if needed.

Reply:
Made a new file imageUploader in utils folder and exported my upload function from there